tendermint-lite/src/main.rs

```diff
@@ -7,35 +7,25 @@ use tendermint::{block::Height, Hash};
 
 use tendermint_lite::{requester::RPCRequester, store::MemStore};
 
-use core::future::Future;
 use std::time::{Duration, SystemTime};
 use tendermint::lite::error::Error;
 use tendermint_lite::store::State;
-use tokio::runtime::Builder;
 
 // TODO: these should be config/args
 static SUBJECTIVE_HEIGHT: u64 = 1;
 static SUBJECTIVE_VALS_HASH_HEX: &str =
     "A5A7DEA707ADE6156F8A981777CA093F178FC790475F6EC659B6617E704871DD";
 static RPC_ADDR: &str = "localhost:26657";
 
-pub fn block_on<F: Future>(future: F) -> F::Output {
-    Builder::new()
-        .basic_scheduler()
-        .enable_all()
-        .build()
-        .unwrap()
-        .block_on(future)
-}
-
-fn main() {
+#[tokio::main]
+async fn main() {
```

Comment:
I can look into incorporating these changes into the abscissa version of this main (in #125) likely with `abscissa_tokio` as @tarcieri suggested.